src/routes/v1/tokens.go

```diff
@@ -0,0 +1,78 @@
+package v1
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/dana-team/platform-backend/src/customerrors"
+	"github.com/dana-team/platform-backend/src/types"
+
+	"github.com/dana-team/platform-backend/src/controllers"
+	"github.com/dana-team/platform-backend/src/middleware"
+	"github.com/gin-gonic/gin"
+)
+
+// tokenHandler wraps a handler function with context setup for tokenController.
+func tokenHandler(handler func(controller controllers.TokenController, c *gin.Context) (interface{}, error)) gin.HandlerFunc {
+	return func(c *gin.Context) {
+		kubeClient, err := middleware.GetKubeClient(c)
+		if middleware.AddErrorToContext(c, err) {
+			return
+		}
+
+		logger, err := middleware.GetLogger(c)
+		if middleware.AddErrorToContext(c, err) {
+			return
+		}
+
+		context := c.Request.Context()
+		tokenController := controllers.NewTokenController(kubeClient, context, logger)
+
+		result, err := handler(tokenController, c)
+		if middleware.AddErrorToContext(c, err) {
+			return
+		}
+
+		c.JSON(http.StatusOK, result)
+	}
+}
+
+// CreateToken returns a Gin handler function for creating a service account token.
+func CreateToken() gin.HandlerFunc {
+	return func(c *gin.Context) {
+		var request types.ServiceAccountRequestUri
+		expirationSeconds := c.DefaultQuery("expirationSeconds", "36000")
```

Comment:
Why is this not in a struct like other query params?

https://github.com/dana-team/platform-backend/blob/42cd557e3785424ba6c893f8274ea22a092f2ad8/src/routes/v1/capp.go#L47-L51